src/relayserver/ServerConfigParams.ts

```diff
@@ -59,6 +60,7 @@ export interface ServerConfigParams {
   defaultGasLimit: number
 
   runPenalizer: boolean
+  runPaymasterReputations: boolean
```

Comment:
this is the only configuration parameter for the reputation system. need to be able to configure others too
(but first, probably need to re-define our configuration system to be hierarchic: too many flat-out params in ServerConfigParams...

Reply:
yeah, the ServerConfigParams, with all due respect, is a mess... I didn't want to dig into it right now...